tests/versioning_3_test.go

```diff
@@ -1464,12 +1456,18 @@ func (s *Versioning3Suite) testChildWorkflowInheritance_ExpectInherit(crossTq bo
 	// v1 is current for both parent and child
 	s.setCurrentDeployment(tv1)
 
-	run, err := s.SdkClient().ExecuteWorkflow(ctx, sdkclient.StartWorkflowOptions{
+	startOpts := sdkclient.StartWorkflowOptions{
 		ID:                  tv1.WorkflowID(),
 		TaskQueue:           tv1.TaskQueue().GetName(),
-		VersioningOverride:  sdkOverride,
+		VersioningOverride:  nil,
```

Comment:
If I make this value be the nil value of `var sdkOverride *sdkclient.PinnedVersioningOverride`, it causes a nil pointer panic [here](https://github.com/temporalio/sdk-go/blob/v1.35.0/internal/internal_workflow_execution_options.go#L137) in the Go SDK. A plain untyped nil as used above works. Is that a problem that should be fixed @Sushisource or is that expected?